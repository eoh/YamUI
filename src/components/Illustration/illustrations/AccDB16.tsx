/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
/* tslint:disable:max-line-length */
// This file was auto-generated -- do not edit

import * as React from 'react';
import BaseIllustration from '../BaseIllustration';

export default class AccDB extends BaseIllustration {
  render() {
    return (
      <svg
        viewBox="0 0 16 16"
        height="16"
        width="16"
        className={this.getClassName()}
        aria-hidden="true"
        tabIndex={-1}
        focusable="false"
      >
        <path fill="#fff" d="M14 15H4V1h8l2 2v12z" />
  <path fill="#949494" d="M12 0H3v16h12V3zm0 1.5L13.5 3H12zM14 15H4V1h7v3h3z" />
  <path fill="#c8c8c8" d="M12 10v3H8v-3h4m1-1H7v5h6V9z" />
  <path fill="#c8c8c8" d="M10 6v3H7V6h3m1-1H6v5h5V5z" />
  <path fill="#a4373a" d="M1 5l7-1v10l-7-1V5z" />
  <path fill="#f7a8ab" d="M8 5h3v2H8zM8 9h5v2H8z" />
  <path fill="#fff" d="M4 9h1v1H4zM4 7h1v1H4zM3 8h1v1H3zM3 9h1v1H3zM5 9h1v1H5zM5 8h1v1H5zM2 10h1v1H2z" />
  <path fill="#fff" d="M3 10h1v1H3zM5 7h1v1H5zM3 7h1v1H3zM2 9h1v1H2zM4 6h1v1H4zM5 10h1v1H5zM6 9h1v1H6z" opacity=".6" />
  <path fill="#fff" d="M6 10h1v1H6z" />
      </svg>
    );
  }
}
