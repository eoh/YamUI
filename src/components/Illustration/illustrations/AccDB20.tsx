/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
/* tslint:disable:max-line-length */
// This file was auto-generated -- do not edit

import * as React from 'react';
import BaseIllustration from '../BaseIllustration';

export default class AccDB extends BaseIllustration {
  render() {
    return (
      <svg
        viewBox="0 0 20 20"
        height="20"
        width="20"
        className={this.getClassName()}
        aria-hidden="true"
        tabIndex={-1}
        focusable="false"
      >
        <path fill="#fff" d="M17 18H5V2h10l2 2v14z" />
  <path fill="#949494" d="M15 1H4v18h14V4zm0 1.5L16.5 4H15zM17 18H5V2h9v3h3z" />
  <path fill="#c8c8c8" d="M15 12v4h-5v-4h5m1-1H9v6h7v-6z" />
  <path fill="#c8c8c8" d="M13 7v4H9V7h4m1-1H8v6h6V6z" />
  <path fill="#a4373a" d="M2 6l8-1v12l-8-1V6z" />
  <path fill="#f7a8ab" d="M10 11h6v2h-6zM10 6h4v2h-4z" />
  <path fill="#fff" d="M7 9V8H5v2h2V9zM8 13v-3H7v2H5v-2H4v3H3v1h2v-1h2v1h2v-1H8z" />
  <path fill="#fff" d="M5 11v1h2v-2H5v1zM7 9h1v1H7zM8 12h1v1H8zM3 12h1v1H3zM4 9h1v1H4z" opacity=".6" />
      </svg>
    );
  }
}
