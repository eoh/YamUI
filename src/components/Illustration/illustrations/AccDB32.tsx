/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
/* tslint:disable:max-line-length */
// This file was auto-generated -- do not edit

import * as React from 'react';
import BaseIllustration from '../BaseIllustration';

export default class AccDB extends BaseIllustration {
  render() {
    return (
      <svg
        viewBox="0 0 32 32"
        height="32"
        width="32"
        className={this.getClassName()}
        aria-hidden="true"
        tabIndex={-1}
        focusable="false"
      >
        <defs>
    <linearGradient id="AccDB32-a" x1="5.084" x2="27.961" y1="3.608" y2="30.872" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".862" stopColor="#5e5e5e" stopOpacity=".063" />
      <stop offset={1} stopOpacity=".1" />
    </linearGradient>
    <linearGradient id="AccDB32-b" x1="22.148" x2="26.114" y1="9.102" y2="5.136" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".446" stopColor="#fff" stopOpacity=".008" />
      <stop offset=".636" stopColor="#fff" stopOpacity=".036" />
      <stop offset=".776" stopColor="#fff" stopOpacity=".085" />
      <stop offset=".892" stopColor="#fff" stopOpacity=".154" />
      <stop offset=".993" stopColor="#fff" stopOpacity=".243" />
      <stop offset={1} stopColor="#fff" stopOpacity=".25" />
    </linearGradient>
    <linearGradient id="AccDB32-c" x1="9.5" x2="9.5" y1="26.438" y2="9.525" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".526" stopColor="#fff" stopOpacity=".007" />
      <stop offset=".715" stopColor="#fff" stopOpacity=".034" />
      <stop offset=".85" stopColor="#fff" stopOpacity=".08" />
      <stop offset=".959" stopColor="#fff" stopOpacity=".146" />
      <stop offset={1} stopColor="#fff" stopOpacity=".18" />
    </linearGradient>
    <linearGradient id="AccDB32-d" x1="11.282" x2="11.617" y1="29.352" y2="26.212" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#828282" stopOpacity={0} />
      <stop offset=".862" stopColor="#303030" stopOpacity=".126" />
      <stop offset={1} stopOpacity=".2" />
    </linearGradient>
    <linearGradient id="AccDB32-e" x1={25} x2={25} y1="10.938" y2="9.058" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#828282" stopOpacity={0} />
      <stop offset=".862" stopColor="#303030" stopOpacity=".063" />
      <stop offset={1} stopOpacity=".1" />
    </linearGradient>
  </defs>
  <path fill="#fff" d="M28 29H8V3h15l5 5v21z" />
  <path fill="#949494" d="M23 2H7v28h22V8zm0 1.5L27.5 8H23zM28 29H8V3h14v6h6z" />
  <path fill="#c8c8c8" d="M25 17v8h-9v-8h9m1-1H15v10h11V16z" />
  <path fill="#c8c8c8" d="M21 12v4h-8v-4h8m1-1H12v6h10v-6z" />
  <path fill="#a4373a" d="M3 11l13-2v18L3 25V11z" />
  <path fill="#fff" d="M14 22h-2.239l-.649-1.78H7.87L7.226 22H5l3.32-8h2.436zm-3.358-3.163l-.98-2.69a3.124 3.124 0 0 1-.152-.719h-.051a2.725 2.725 0 0 1-.16.697l-.992 2.712z" />
  <path fill="#f7a8ab" d="M16 16h10v2H16zM16 11h6v2h-6zM17 21h2v1h-2z" />
  <path fill="#c8c8c8" d="M20 21h4v1h-4z" />
  <path fill="#f7a8ab" d="M17 19h2v1h-2z" />
  <path fill="#c8c8c8" d="M20 19h4v1h-4zM17 14h3v1h-3z" />
  <path fill="#f7a8ab" d="M17 23h2v1h-2z" />
  <path fill="#c8c8c8" d="M20 23h4v1h-4z" />
  <path fill="url(#AccDB32-a)" d="M29 8l-6-6H7v28h22z" />
  <path fill="url(#AccDB32-b)" d="M7 2v1h15v6h6v15.5h1V8l-6-6H7z" />
  <path fill="url(#AccDB32-c)" d="M16 9L3 11v14l13 2V9z" />
  <path fill="url(#AccDB32-d)" d="M16 29H7v-3.38L16 27v2z" />
  <path fill="url(#AccDB32-e)" d="M22 9h6v2h-6z" />
      </svg>
    );
  }
}
