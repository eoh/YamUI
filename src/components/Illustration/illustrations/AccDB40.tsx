/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
/* tslint:disable:max-line-length */
// This file was auto-generated -- do not edit

import * as React from 'react';
import BaseIllustration from '../BaseIllustration';

export default class AccDB extends BaseIllustration {
  render() {
    return (
      <svg
        viewBox="0 0 40 40"
        height="40"
        width="40"
        className={this.getClassName()}
        aria-hidden="true"
        tabIndex={-1}
        focusable="false"
      >
        <defs>
    <linearGradient id="AccDB40-a" x1="6.099" x2="35.395" y1="4.434" y2="39.347" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".862" stopColor="#5e5e5e" stopOpacity=".063" />
      <stop offset={1} stopOpacity=".1" />
    </linearGradient>
    <linearGradient id="AccDB40-b" x1={32} x2={32} y1="13.906" y2="11.088" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#828282" stopOpacity={0} />
      <stop offset=".862" stopColor="#303030" stopOpacity=".063" />
      <stop offset={1} stopOpacity=".1" />
    </linearGradient>
    <linearGradient id="AccDB40-c" x1="26.792" x2="31.73" y1="9.708" y2="4.77" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".446" stopColor="#fff" stopOpacity=".008" />
      <stop offset=".636" stopColor="#fff" stopOpacity=".036" />
      <stop offset=".776" stopColor="#fff" stopOpacity=".085" />
      <stop offset=".892" stopColor="#fff" stopOpacity=".154" />
      <stop offset=".993" stopColor="#fff" stopOpacity=".243" />
      <stop offset={1} stopColor="#fff" stopOpacity=".25" />
    </linearGradient>
    <linearGradient id="AccDB40-d" x1="14.278" x2="14.657" y1="36.941" y2="33.377" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#828282" stopOpacity={0} />
      <stop offset=".997" stopOpacity=".2" />
    </linearGradient>
    <linearGradient id="AccDB40-e" x1="11.5" x2="11.5" y1="32.792" y2="12.217" gradientUnits="userSpaceOnUse">
      <stop offset={0} stopColor="#fff" stopOpacity={0} />
      <stop offset=".526" stopColor="#fff" stopOpacity=".007" />
      <stop offset=".715" stopColor="#fff" stopOpacity=".034" />
      <stop offset=".85" stopColor="#fff" stopOpacity=".08" />
      <stop offset=".959" stopColor="#fff" stopOpacity=".146" />
      <stop offset={1} stopColor="#fff" stopOpacity=".18" />
    </linearGradient>
  </defs>
  <path fill="#fff" d="M10 3h19l7 7v27H10V3z" />
  <path fill="#949494" d="M29 2H9v36h28V10zm0 1.5l6.5 6.5H29zM36 37H10V3h18v8h8z" />
  <path fill="#c8c8c8" d="M32 21v11H20V21h12m1-1H19v13h14V20z" />
  <path fill="#c8c8c8" d="M27 15v6H16v-6h11m1-1H15v8h13v-8z" />
  <path fill="#a4373a" d="M3 13l17-2v23L3 32V13z" />
  <path fill="#fff" d="M16 27h-2.257l-.654-2.002H9.82L9.173 27H6.928l3.347-9h2.455zm-3.385-3.559l-.987-3.025a3.849 3.849 0 0 1-.154-.81h-.052a3.38 3.38 0 0 1-.16.785l-1 3.05z" />
  <path fill="#f7a8ab" d="M22 24h2v1h-2zM20 20h13v3H20zM20 14h8v3h-8z" />
  <path fill="#c8c8c8" d="M26 24h4v1h-4zM26 27h4v1h-4zM21 18h4v1h-4z" />
  <path fill="#f7a8ab" d="M22 27h2v1h-2z" />
  <path fill="#c8c8c8" d="M26 30h4v1h-4z" />
  <path fill="#f7a8ab" d="M22 30h2v1h-2z" />
  <path fill="url(#AccDB40-a)" d="M29 2H9v10.294L3 13v19l6 .706V38h28V10z" />
  <path fill="url(#AccDB40-b)" d="M28 11h8v3h-8z" />
  <path fill="url(#AccDB40-c)" d="M29 2H9v1h19v8h8v14h1V10l-8-8z" />
  <path fill="url(#AccDB40-d)" d="M20 37H9v-4.29L20 34v3z" />
  <path fill="url(#AccDB40-e)" d="M20 34L3 32V13l17-2v23z" />
      </svg>
    );
  }
}
