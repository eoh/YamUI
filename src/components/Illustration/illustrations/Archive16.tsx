/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
/* tslint:disable:max-line-length */
// This file was auto-generated -- do not edit

import * as React from 'react';
import BaseIllustration from '../BaseIllustration';

export default class Archive extends BaseIllustration {
  render() {
    return (
      <svg
        viewBox="0 0 16 16"
        height="16"
        width="16"
        className={this.getClassName()}
        aria-hidden="true"
        tabIndex={-1}
        focusable="false"
      >
        <path fill="#a6a6a6" d="M0 14V4h7l2-2h7v12H0z" />
  <path fill="#666" d="M9 2L7 4l2 2h7V2H9z" />
  <path fill="#eaeaea" d="M2 4h2v1H2zM2 12h2v2H2zM1 6v5h4V6zm3 1v3H2V7z" />
      </svg>
    );
  }
}
