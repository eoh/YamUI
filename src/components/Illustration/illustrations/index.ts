/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */

export { default as accdb16 } from './accdb16';
export { default as archive16 } from './archive16';
export { default as audio16 } from './audio16';
export { default as audiowatermark16 } from './audiowatermark16';
export { default as code16 } from './code16';
export { default as csv16 } from './csv16';
export { default as docset16 } from './docset16';
export { default as docx16 } from './docx16';
export { default as dotx16 } from './dotx16';
export { default as email16 } from './email16';
export { default as exe16 } from './exe16';
export { default as folder16 } from './folder16';
export { default as font16 } from './font16';
export { default as genericfile16 } from './genericfile16';
export { default as html16 } from './html16';
export { default as link16 } from './link16';
export { default as listitem16 } from './listitem16';
export { default as model16 } from './model16';
export { default as mpp16 } from './mpp16';
export { default as mpt16 } from './mpt16';
export { default as multiple16 } from './multiple16';
export { default as odp16 } from './odp16';
export { default as ods16 } from './ods16';
export { default as odt16 } from './odt16';
export { default as one16 } from './one16';
export { default as onetoc16 } from './onetoc16';
export { default as pdf16 } from './pdf16';
export { default as photo16 } from './photo16';
export { default as photowatermark16 } from './photowatermark16';
export { default as potx16 } from './potx16';
export { default as ppsx16 } from './ppsx16';
export { default as pptx16 } from './pptx16';
export { default as presentation16 } from './presentation16';
export { default as pub16 } from './pub16';
export { default as rtf16 } from './rtf16';
export { default as sharedfolder16 } from './sharedfolder16';
export { default as spo16 } from './spo16';
export { default as spreadsheet16 } from './spreadsheet16';
export { default as sway16 } from './sway16';
export { default as sysfile16 } from './sysfile16';
export { default as txt16 } from './txt16';
export { default as vector16 } from './vector16';
export { default as video16 } from './video16';
export { default as videowatermark16 } from './videowatermark16';
export { default as vsdx16 } from './vsdx16';
export { default as vssx16 } from './vssx16';
export { default as vstx16 } from './vstx16';
export { default as xlsx16 } from './xlsx16';
export { default as xltx16 } from './xltx16';
export { default as xml16 } from './xml16';
export { default as xsn16 } from './xsn16';
export { default as zip16 } from './zip16';
export { default as accdb20 } from './accdb20';
export { default as archive20 } from './archive20';
export { default as audio20 } from './audio20';
export { default as audiowatermark20 } from './audiowatermark20';
export { default as code20 } from './code20';
export { default as csv20 } from './csv20';
export { default as docset20 } from './docset20';
export { default as docx20 } from './docx20';
export { default as dotx20 } from './dotx20';
export { default as email20 } from './email20';
export { default as exe20 } from './exe20';
export { default as folder20 } from './folder20';
export { default as font20 } from './font20';
export { default as genericfile20 } from './genericfile20';
export { default as html20 } from './html20';
export { default as link20 } from './link20';
export { default as listitem20 } from './listitem20';
export { default as model20 } from './model20';
export { default as mpp20 } from './mpp20';
export { default as mpt20 } from './mpt20';
export { default as multiple20 } from './multiple20';
export { default as odp20 } from './odp20';
export { default as ods20 } from './ods20';
export { default as odt20 } from './odt20';
export { default as one20 } from './one20';
export { default as onetoc20 } from './onetoc20';
export { default as pdf20 } from './pdf20';
export { default as photo20 } from './photo20';
export { default as photowatermark20 } from './photowatermark20';
export { default as potx20 } from './potx20';
export { default as ppsx20 } from './ppsx20';
export { default as pptx20 } from './pptx20';
export { default as presentation20 } from './presentation20';
export { default as pub20 } from './pub20';
export { default as rtf20 } from './rtf20';
export { default as sharedfolder20 } from './sharedfolder20';
export { default as spo20 } from './spo20';
export { default as spreadsheet20 } from './spreadsheet20';
export { default as sway20 } from './sway20';
export { default as sysfile20 } from './sysfile20';
export { default as txt20 } from './txt20';
export { default as vector20 } from './vector20';
export { default as video20 } from './video20';
export { default as videowatermark20 } from './videowatermark20';
export { default as vsdx20 } from './vsdx20';
export { default as vssx20 } from './vssx20';
export { default as vstx20 } from './vstx20';
export { default as xlsx20 } from './xlsx20';
export { default as xltx20 } from './xltx20';
export { default as xml20 } from './xml20';
export { default as xsn20 } from './xsn20';
export { default as zip20 } from './zip20';
export { default as accdb32 } from './accdb32';
export { default as archive32 } from './archive32';
export { default as audio32 } from './audio32';
export { default as audiowatermark32 } from './audiowatermark32';
export { default as code32 } from './code32';
export { default as csv32 } from './csv32';
export { default as docset32 } from './docset32';
export { default as docx32 } from './docx32';
export { default as dotx32 } from './dotx32';
export { default as email32 } from './email32';
export { default as exe32 } from './exe32';
export { default as folder32 } from './folder32';
export { default as font32 } from './font32';
export { default as genericfile32 } from './genericfile32';
export { default as html32 } from './html32';
export { default as link32 } from './link32';
export { default as listitem32 } from './listitem32';
export { default as model32 } from './model32';
export { default as mpp32 } from './mpp32';
export { default as mpt32 } from './mpt32';
export { default as multiple32 } from './multiple32';
export { default as odp32 } from './odp32';
export { default as ods32 } from './ods32';
export { default as odt32 } from './odt32';
export { default as one32 } from './one32';
export { default as onetoc32 } from './onetoc32';
export { default as pdf32 } from './pdf32';
export { default as photo32 } from './photo32';
export { default as photowatermark32 } from './photowatermark32';
export { default as potx32 } from './potx32';
export { default as ppsx32 } from './ppsx32';
export { default as pptx32 } from './pptx32';
export { default as presentation32 } from './presentation32';
export { default as pub32 } from './pub32';
export { default as rtf32 } from './rtf32';
export { default as sharedfolder32 } from './sharedfolder32';
export { default as spo32 } from './spo32';
export { default as spreadsheet32 } from './spreadsheet32';
export { default as sway32 } from './sway32';
export { default as sysfile32 } from './sysfile32';
export { default as txt32 } from './txt32';
export { default as vector32 } from './vector32';
export { default as video32 } from './video32';
export { default as videowatermark32 } from './videowatermark32';
export { default as vsdx32 } from './vsdx32';
export { default as vssx32 } from './vssx32';
export { default as vstx32 } from './vstx32';
export { default as xlsx32 } from './xlsx32';
export { default as xltx32 } from './xltx32';
export { default as xml32 } from './xml32';
export { default as xsn32 } from './xsn32';
export { default as zip32 } from './zip32';
export { default as accdb40 } from './accdb40';
export { default as archive40 } from './archive40';
export { default as audio40 } from './audio40';
export { default as audiowatermark40 } from './audiowatermark40';
export { default as code40 } from './code40';
export { default as csv40 } from './csv40';
export { default as docset40 } from './docset40';
export { default as docx40 } from './docx40';
export { default as dotx40 } from './dotx40';
export { default as email40 } from './email40';
export { default as exe40 } from './exe40';
export { default as folder40 } from './folder40';
export { default as font40 } from './font40';
export { default as genericfile40 } from './genericfile40';
export { default as html40 } from './html40';
export { default as link40 } from './link40';
export { default as listitem40 } from './listitem40';
export { default as model40 } from './model40';
export { default as mpp40 } from './mpp40';
export { default as mpt40 } from './mpt40';
export { default as multiple40 } from './multiple40';
export { default as odp40 } from './odp40';
export { default as ods40 } from './ods40';
export { default as odt40 } from './odt40';
export { default as one40 } from './one40';
export { default as onetoc40 } from './onetoc40';
export { default as pdf40 } from './pdf40';
export { default as photo40 } from './photo40';
export { default as photowatermark40 } from './photowatermark40';
export { default as potx40 } from './potx40';
export { default as ppsx40 } from './ppsx40';
export { default as pptx40 } from './pptx40';
export { default as presentation40 } from './presentation40';
export { default as pub40 } from './pub40';
export { default as rtf40 } from './rtf40';
export { default as sharedfolder40 } from './sharedfolder40';
export { default as spo40 } from './spo40';
export { default as spreadsheet40 } from './spreadsheet40';
export { default as sway40 } from './sway40';
export { default as sysfile40 } from './sysfile40';
export { default as txt40 } from './txt40';
export { default as vector40 } from './vector40';
export { default as video40 } from './video40';
export { default as videowatermark40 } from './videowatermark40';
export { default as vsdx40 } from './vsdx40';
export { default as vssx40 } from './vssx40';
export { default as vstx40 } from './vstx40';
export { default as xlsx40 } from './xlsx40';
export { default as xltx40 } from './xltx40';
export { default as xml40 } from './xml40';
export { default as xsn40 } from './xsn40';
export { default as zip40 } from './zip40';
export { default as accdb48 } from './accdb48';
export { default as archive48 } from './archive48';
export { default as audio48 } from './audio48';
export { default as audiowatermark48 } from './audiowatermark48';
export { default as code48 } from './code48';
export { default as csv48 } from './csv48';
export { default as docset48 } from './docset48';
export { default as docx48 } from './docx48';
export { default as dotx48 } from './dotx48';
export { default as email48 } from './email48';
export { default as exe48 } from './exe48';
export { default as folder48 } from './folder48';
export { default as font48 } from './font48';
export { default as genericfile48 } from './genericfile48';
export { default as html48 } from './html48';
export { default as link48 } from './link48';
export { default as listitem48 } from './listitem48';
export { default as model48 } from './model48';
export { default as mpp48 } from './mpp48';
export { default as mpt48 } from './mpt48';
export { default as multiple48 } from './multiple48';
export { default as odp48 } from './odp48';
export { default as ods48 } from './ods48';
export { default as odt48 } from './odt48';
export { default as one48 } from './one48';
export { default as onetoc48 } from './onetoc48';
export { default as pdf48 } from './pdf48';
export { default as photo48 } from './photo48';
export { default as photowatermark48 } from './photowatermark48';
export { default as potx48 } from './potx48';
export { default as ppsx48 } from './ppsx48';
export { default as pptx48 } from './pptx48';
export { default as presentation48 } from './presentation48';
export { default as pub48 } from './pub48';
export { default as rtf48 } from './rtf48';
export { default as sharedfolder48 } from './sharedfolder48';
export { default as spo48 } from './spo48';
export { default as spreadsheet48 } from './spreadsheet48';
export { default as sway48 } from './sway48';
export { default as sysfile48 } from './sysfile48';
export { default as txt48 } from './txt48';
export { default as vector48 } from './vector48';
export { default as video48 } from './video48';
export { default as videowatermark48 } from './videowatermark48';
export { default as vsdx48 } from './vsdx48';
export { default as vssx48 } from './vssx48';
export { default as vstx48 } from './vstx48';
export { default as xlsx48 } from './xlsx48';
export { default as xltx48 } from './xltx48';
export { default as xml48 } from './xml48';
export { default as xsn48 } from './xsn48';
export { default as zip48 } from './zip48';
export { default as accdb72 } from './accdb72';
export { default as archive72 } from './archive72';
export { default as audio72 } from './audio72';
export { default as audiowatermark72 } from './audiowatermark72';
export { default as code72 } from './code72';
export { default as csv72 } from './csv72';
export { default as docset72 } from './docset72';
export { default as docx72 } from './docx72';
export { default as dotx72 } from './dotx72';
export { default as email72 } from './email72';
export { default as exe72 } from './exe72';
export { default as folder72 } from './folder72';
export { default as font72 } from './font72';
export { default as genericfile72 } from './genericfile72';
export { default as html72 } from './html72';
export { default as link72 } from './link72';
export { default as listitem72 } from './listitem72';
export { default as model72 } from './model72';
export { default as mpp72 } from './mpp72';
export { default as mpt72 } from './mpt72';
export { default as multiple72 } from './multiple72';
export { default as odp72 } from './odp72';
export { default as ods72 } from './ods72';
export { default as odt72 } from './odt72';
export { default as one72 } from './one72';
export { default as onetoc72 } from './onetoc72';
export { default as pdf72 } from './pdf72';
export { default as photo72 } from './photo72';
export { default as photowatermark72 } from './photowatermark72';
export { default as potx72 } from './potx72';
export { default as ppsx72 } from './ppsx72';
export { default as pptx72 } from './pptx72';
export { default as presentation72 } from './presentation72';
export { default as pub72 } from './pub72';
export { default as rtf72 } from './rtf72';
export { default as sharedfolder72 } from './sharedfolder72';
export { default as spo72 } from './spo72';
export { default as spreadsheet72 } from './spreadsheet72';
export { default as sway72 } from './sway72';
export { default as sysfile72 } from './sysfile72';
export { default as txt72 } from './txt72';
export { default as vector72 } from './vector72';
export { default as video72 } from './video72';
export { default as videowatermark72 } from './videowatermark72';
export { default as vsdx72 } from './vsdx72';
export { default as vssx72 } from './vssx72';
export { default as vstx72 } from './vstx72';
export { default as xlsx72 } from './xlsx72';
export { default as xltx72 } from './xltx72';
export { default as xml72 } from './xml72';
export { default as xsn72 } from './xsn72';
export { default as zip72 } from './zip72';
