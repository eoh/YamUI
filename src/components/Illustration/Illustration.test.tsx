/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
import * as React from 'react';
import { shallow, ShallowWrapper } from 'enzyme';
import { BaseIllustration, IllustrationProps } from './index';
import * as allExportedIllustrations from './illustrations';
import AccDB16 from './illustrations/AccDB16';

describe('<Illustration />', () => {
  let component: ShallowWrapper<IllustrationProps, {}>;

  describe('an illustration', () => {
    describe('with minimal options', () => {
      beforeEach(() => {
        component = shallow(<AccDB16 />);
      });

      it('matches its snapshot', () => {
        expect(component).toMatchSnapshot();
      });
    });

    describe('with block', () => {
      beforeEach(() => {
        component = shallow(<AccDB16 block={true} />);
      });

      it('matches its snapshot', () => {
        expect(component).toMatchSnapshot();
      });
    });

    describe('with additional className', () => {
      beforeEach(() => {
        component = shallow(<AccDB16 className="TEST_CLASSNAME" />);
      });

      it('matches its snapshot', () => {
        expect(component).toMatchSnapshot();
      });
    });
  });

  describe('total number of illustrations', () => {
    const totalIcons = Object.keys(allExportedIllustrations).length;
    it('is correct', () => {
      expect(totalIcons).toBe(312);
    });
  });

  describe('each illustration', () => {
    let Illustration: typeof BaseIllustration;

    Object.keys(allExportedIllustrations).forEach((name) => {
      beforeEach(() => {
        Illustration = (allExportedIllustrations as any)[name];
        component = shallow(<Illustration />);
      });

      it('has a viewbox', () => {
        expect(component.getNode().props.viewBox).toBeDefined();
      });
    });
  });
});
