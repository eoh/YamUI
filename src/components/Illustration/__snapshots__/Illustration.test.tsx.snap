// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Illustration /> an illustration with additional className matches its snapshot 1`] = `
<svg
  aria-hidden="true"
  className="y-illustration TEST_CLASSNAME"
  focusable="false"
  height="16"
  tabIndex={-1}
  viewBox="0 0 16 16"
  width="16"
>
  <path
    d="M14 15H4V1h8l2 2z"
    fill="#FFF"
  />
  <path
    d="M12 0H3v16h12V3l-3-3zm0 1.5L13.5 3H12V1.5zM14 15H4V1h7v3h3v11z"
    fill="#949494"
  />
  <path
    d="M12 10v3H8v-3h4m1-1H7v5h6V9z"
    fill="#C8C8C8"
  />
  <path
    d="M10 6v3H7V6h3m1-1H6v5h5V5z"
    fill="#C8C8C8"
  />
  <path
    d="M1 5l7-1v10l-7-1z"
    fill="#A4373A"
  />
  <path
    d="M8 5h3v2H8zM8 9h5v2H8z"
    fill="#F7A8AB"
  />
  <g
    fill="#FFF"
  >
    <path
      d="M4 9h1v1H4zM4 7h1v1H4zM3 8h1v1H3zM3 9h1v1H3zM5 9h1v1H5zM5 8h1v1H5zM2 10h1v1H2z"
    />
    <path
      d="M3 10h1v1H3zM5 7h1v1H5zM3 7h1v1H3zM2 9h1v1H2zM4 6h1v1H4zM5 10h1v1H5zM6 9h1v1H6z"
      opacity=".6"
    />
    <path
      d="M6 10h1v1H6z"
    />
  </g>
</svg>
`;

exports[`<Illustration /> an illustration with block matches its snapshot 1`] = `
<svg
  aria-hidden="true"
  className="y-illustration y-illustration__isBlock"
  focusable="false"
  height="16"
  tabIndex={-1}
  viewBox="0 0 16 16"
  width="16"
>
  <path
    d="M14 15H4V1h8l2 2z"
    fill="#FFF"
  />
  <path
    d="M12 0H3v16h12V3l-3-3zm0 1.5L13.5 3H12V1.5zM14 15H4V1h7v3h3v11z"
    fill="#949494"
  />
  <path
    d="M12 10v3H8v-3h4m1-1H7v5h6V9z"
    fill="#C8C8C8"
  />
  <path
    d="M10 6v3H7V6h3m1-1H6v5h5V5z"
    fill="#C8C8C8"
  />
  <path
    d="M1 5l7-1v10l-7-1z"
    fill="#A4373A"
  />
  <path
    d="M8 5h3v2H8zM8 9h5v2H8z"
    fill="#F7A8AB"
  />
  <g
    fill="#FFF"
  >
    <path
      d="M4 9h1v1H4zM4 7h1v1H4zM3 8h1v1H3zM3 9h1v1H3zM5 9h1v1H5zM5 8h1v1H5zM2 10h1v1H2z"
    />
    <path
      d="M3 10h1v1H3zM5 7h1v1H5zM3 7h1v1H3zM2 9h1v1H2zM4 6h1v1H4zM5 10h1v1H5zM6 9h1v1H6z"
      opacity=".6"
    />
    <path
      d="M6 10h1v1H6z"
    />
  </g>
</svg>
`;

exports[`<Illustration /> an illustration with minimal options matches its snapshot 1`] = `
<svg
  aria-hidden="true"
  className="y-illustration"
  focusable="false"
  height="16"
  tabIndex={-1}
  viewBox="0 0 16 16"
  width="16"
>
  <path
    d="M14 15H4V1h8l2 2z"
    fill="#FFF"
  />
  <path
    d="M12 0H3v16h12V3l-3-3zm0 1.5L13.5 3H12V1.5zM14 15H4V1h7v3h3v11z"
    fill="#949494"
  />
  <path
    d="M12 10v3H8v-3h4m1-1H7v5h6V9z"
    fill="#C8C8C8"
  />
  <path
    d="M10 6v3H7V6h3m1-1H6v5h5V5z"
    fill="#C8C8C8"
  />
  <path
    d="M1 5l7-1v10l-7-1z"
    fill="#A4373A"
  />
  <path
    d="M8 5h3v2H8zM8 9h5v2H8z"
    fill="#F7A8AB"
  />
  <g
    fill="#FFF"
  >
    <path
      d="M4 9h1v1H4zM4 7h1v1H4zM3 8h1v1H3zM3 9h1v1H3zM5 9h1v1H5zM5 8h1v1H5zM2 10h1v1H2z"
    />
    <path
      d="M3 10h1v1H3zM5 7h1v1H5zM3 7h1v1H3zM2 9h1v1H2zM4 6h1v1H4zM5 10h1v1H5zM6 9h1v1H6z"
      opacity=".6"
    />
    <path
      d="M6 10h1v1H6z"
    />
  </g>
</svg>
`;
